github/PullRequest.py

```diff
@@ -467,12 +467,39 @@ def create_review_comment(
         in_reply_to: Opt[int] = NotSet,
         subject_type: Opt[str] = NotSet,
         as_suggestion: bool = False,
+    ) -> PullRequestComment:
+        return self.create_review_comment_with_primatives(
+            body=body,
+            commit=commit._identity,
+            path=path,
+            line=line,
+            side=side,
+            start_line=start_line,
+            start_side=start_side,
+            in_reply_to=in_reply_to,
+            subject_type=subject_type,
+            as_suggestion=as_suggestion,
+        )
+
+    def create_review_comment_with_primatives(
+        self,
+        body: str,
+        commit: str,
```

Comment:
I'd rather go for one method with `commit: Commit | str`.

Reply:
Updated :)